Styles/components.css

```diff
@@ -58,11 +57,13 @@ body {
   padding: 0.3em;
   box-shadow: none;
   cursor: pointer;
+  background-color: transparent;
+  color: inherit;
 }
-.alert-white {
-  color: black;
-  border-color: white;
-  background-color: white;
+.alert-mycolor {
```

Comment:
The text color contrast in _alert_ boxes isn't balanced. The texts are almost invisible. Try fixing the color.